### PR TITLE
Ks mu sig schnorr sig

### PR DESCRIPTION
**Pull request by Kevoulee**

### CODE REVIEW

**Comment by Kevoulee** [comment]
Hi @hansieodendaal, please see the updated report. I have attached your pdf with my checks to your review comments 
[The_MuSig_Schnorr_Signature_Scheme.Hansie.pdf](https://github.com/tari-labs/tari-university/files/2759357/The_MuSig_Schnorr_Signature_Scheme.Hansie.pdf)

**Comment by CjS77** [comment]
Hey @Kevoulee. `.lyx` files aren't an open standard.. Can you also export your report in `.tex` format (LyX must offer this) so that people can use normal LaTex to build your reports?

File: src/cryptography/musig-schnorr-sig-scheme/The_MuSig_Schnorr_Signature_Scheme.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+
+The above described is referred to as the so-called “key-prefixed” variant of the scheme, which sees the public key hashed together with \(R\) and \(m\). This variant was thought to have a better multi-user security bound than the classic variant , however in  the key-prefixing was seen as unnecessary to enable good multi-user security for Schnorr signatures.
+
+For the development of the new Schnorr-based multi-signature scheme, key-prefixing seemed a requirement for the security proof to go through, despite not knowing the form of an attack. The rationale also follows the process in reality, as messages signed in Bitcoin always indirectly commits to the public key.
+
+### Rogue attacks 
+
+Rogue attacks are a significant concern when implementing multi-signature schemes. Here a subset of corrupted singers, manipulate the public keys computed as functions of the public keys of honest users, allowing them to easily produce forgeries for the set of public keys (despite them not knowing the associated secret keys).
```
Comment (CjS77) [inline src/cryptography/musig-schnorr-sig-scheme/The_MuSig_Schnorr_Signature_Scheme.md]: Damn those corrupted singers. Miley Cyrus being the worst of the lot.

Comment (CjS77) [reply src/cryptography/musig-schnorr-sig-scheme/The_MuSig_Schnorr_Signature_Scheme.md]: BTW, you can link to the key cancellation attack demo in the Into to Digital Signatures talk as a great example of this

**Comment by CjS77** [comment]
One suggestion to make reviewing easier (you don't have to do it now, but for next time) is to set a hard line break at 120 characters, which makes line comments more granular.

**Comment by Kevoulee** [comment]
@hansieodendaal please see converted .md file. Please let me know changes. Many thanks

**Comment by Kevoulee** [comment]
@hansieodendaal I have removed a section of formation of MuSig. I hope this is better now. Thank you.

**Comment by Kevoulee** [comment]
@hansieodendaal please see changes to formulae and reordering. Thank you so much.

**Comment by hansieodendaal** [comment]
Hi Kev,

Thanks, looks great at birds eye view. Some specific comments:

- [ ] Please resolve conflicts in README.md

- [ ] Please update Contents with the new reordered layout and test llinks

- [ ] See editorial edits below

**Pull request merged**